### DISCUSSION TITLE
流量用完，樱花有没有包月服务和一些小问题。

### DISCUSSION THREAD

**Discussion by xiaokai0309** [natfrp/wiki#115] [水深火热]
多模组服流量消耗太大，一天7~8G光流量就是个不小的开支。
还有就是我自己有一个闲置的电脑同时还有一个闲置的vps，有什么办法让我服务器开在本地，然后使用vps的外网ip？

**Comment by expliyh** [comment]
自己用开源的frp搭建就行了
